test/connections.test.js

```diff
@@ -0,0 +1,9 @@
+import { describe, expect, it } from 'vitest';
+import { connectToDB } from '../connections.js';
+
+describe('checks db connection function', () => {
+  it('Should connect to DB', async () => {
+    const res = await connectToDB();
+    expect(res).toBeTruthy();
```

Comment:
You can add "ping" test if you like (why?)

Reply:
added